pkgs/by-name/pi/pingtunnel/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "pingtunnel";
+  version = "2.8";
+
+  src = fetchFromGitHub {
+    owner = "esrrhs";
+    repo = "pingtunnel";
+    rev = version;
+    hash = "sha256-qkIeE6gNBUUd5/IhcKhDmXZbzeSTR0r6qCAK/Nz7Kew=";
+  };
+
+  vendorHash = "sha256-SkBHFvQn3Q6fl/d1npEspTYhq1YrnusXNWr/Ky4bv64=";
+
+  ldflags = [ "-s" "-w" ];
+
+  postInstall = "mv $out/bin/cmd $out/bin/pingtunnel";
+
+  meta = with lib; {
+    description = "A tool that send TCP/UDP traffic over ICMP";
```

Comment:
```suggestion
    description = "Tool that send TCP/UDP traffic over ICMP";
```

Reply:
Isn't this quite usual?

Reply:
The contributing guide states that we should remove those IIRC